packages/xo-web/src/xo-app/vm-import/index.js

```diff
@@ -259,28 +259,32 @@ export default class Import extends Component {
     this.setState({
       vms: [],
     })
+    document.body.style.cursor = 'wait'
```

Comment:
Wouldn't that be better to save the previous value and reset it at the end?

Reply:
I removed it altogether, I think it's not really necessary